setup.py

```diff
@@ -32,7 +32,7 @@
 
     install_requirements = [
         "numpy",
-        "pandas<2.1.0",
+        "pandas",
```

Comment:
Would it hurt to make this >2.1.0?

Reply:
I was a little wary to do this, since `pip` was having trouble in US CVD with resolving all the dependencies out of the box with pandas bounded from below (in CVD's `setup.py`, not `vivarium`), and I didn't want to break anything for anybody. But I guess with vivarium's pandas bounded from below, it's still fine, since pip can just install the release before this.

Reply:
Correct me if I'm wrong, but I think the way you have it implemented, things should still work in `pandas<2.1.0`. In that case pinning a lower bound is not correct (i.e. it is correct as you have it).

Reply:
yes, it is backwards compatible